### ISSUE TITLE
blurr image is resultant image . how to increase its resolution?

### ISSUE THREAD

**Issue by apurvaaggarwal** [boxme/SquareCamera#52]

**Comment by Codeglee** [comment]
I think this means, when you're using SquareCamera on a device with a higher resolution (tablet for example) the preview window shows a blurry / pixelated view until you capture the photo.

**Comment by apurvaaggarwal** [comment]
Yes,do u know its solution .so ,that I get a clear picture like a normal camera
